src/style/typography.scss

```diff
@@ -5,8 +5,8 @@
 
 /// Constants ------------------------------------------------------------------
 
-$iui-sans: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
-$iui-monospace: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+$iui-sans: 'Open Sans', BlinkMacSystemFont, -apple-system, 'Segoe UI', Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', sans-serif;
```

Comment:
Are `'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'` really necessary?

Reply:
I'll remove them.  CSS-Tricks recommended the stack, I mostly just copy / pasted.  I will say, I have not tested emojis in our pages at all.